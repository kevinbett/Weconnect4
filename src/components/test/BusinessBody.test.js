import RenderBusiness from '../BusinessBody'
import { shallow } from "enzyme"

describe('Business Body',() => { 
    it ('Business body renders', () => { 
        expect(shallow(<RenderBusiness/>)).toMatchSnapshot()
    });
} )