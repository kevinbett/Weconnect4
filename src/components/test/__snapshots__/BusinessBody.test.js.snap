// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business Body Business body renders 1`] = `
<div
  className="container"
>
  <div
    className="row"
  />
</div>
`;
